numpy/core/tests/test_ufunc.py

```diff
@@ -1705,11 +1705,11 @@ def test_initial_reduction(self):
         res = np.add.reduce(a, initial=5)
         assert_equal(res, 15)
 
-    def test_empty_reduction_and_idenity(self):
+    def test_empty_reduction_and_identity(self):
```

Comment:
Not sure about this one, `idenity` is not a word, is it?

Reply:
Looks good.